TestFoundation/TestHost.swift

```diff
@@ -32,5 +34,17 @@ class TestHost: XCTestCase {
         let swiftAddressesSecond = swift.addresses
         XCTAssertEqual(swiftAddressesSecond.count, swiftAddressesFirst.count)
     }
+
+    func test_isEqual_positive() {
+        let host0 = Host(address: "8.8.8.8")
+        let host1 = Host(address: "8.8.8.8")
+        XCTAssertTrue(host0.isEqual(to: host1))
+    }
+
+    func test_isEqual_negative() {
+        let host0 = Host(address: "8.8.8.8")
+        let host1 = Host(address: "8.8.8.9")
+        XCTAssertFalse(host0.isEqual(to: host1))
+    }
 }
```

Comment:
I would just write these as one test (`test_isEqual`) with the  `XCTAssertTrue` and `XCTAssertFalse`. Thanks

Reply:
Thank you for watching.
I fixed that.

Reply:
Is there any guideline on how to structure the tests?

I usually prefer to make them smaller with only one goal in mind. So in this case one is the positive (isEqual), the other the negative test (not isEqual).

Reply:
In general I would use 1 test function per method that is being tested, testing both positive and negative values as required. Seeing the whole test function fail to pass will indicate that there is an issue with the method implementation.

Using 1 function per test will just lead to an excess of functions. Bear in mind that some types/classes do not have much functionality at all so may only require a few test functions, whereas large complex types, eg `JSONSerialization` require lots of tests for each method.

Reply:
Thank you very much for your answer. Gonna update the PR #2035 then accordingly.